_layouts/category_archives.html

```diff
@@ -3,7 +3,16 @@
 ---
 <div class="home">
 
-  <h1 class="page-heading">Archive of {{ page.type }} '{{ page.title | escape }}'</h1>
+  {%- capture i18n-tag -%}
+    archive-{{ page.type }}
+  {% endcapture %}
```

Comment:
How about this? This won't require stripping. The same for `capture`s below.

```suggestion
  {% capture i18n-tag %}archive-{{ page.type }}{% endcapture %}
```